docs/source/gbd2017_models/causes/smoking_mortality/index.rst

```diff
@@ -0,0 +1,457 @@
+
+	The :ref:`Vivarium CSU Lung Cancer Screening  Model  <2017_concept_model_vivarium_swissre_lungcancer>` only models simulants 15 years (age group ID 8) and older, so notes were made for causes with age group starts older than 15 years only.
+
+Restrictions
+++++++++++++
+
+See the table above.
+
+Vivarium Modeling Strategy
+--------------------------
+
+Scope
++++++
+
+The Vivarium modeling strategy outlined in this document will be a **mortality only** model that combines several causes within GBD. This cause is intended to be modeled as a outcome pair with the smoking risk factor.
+
+Rather than individual measure the relative risk of mortality due to each individual smoking affected cause to create an overall summary relative risk of mortality due to smoking affected causes, this model will use the all cause mortality PAF for smoking, subtract out the contribution of lung cancer, and then back-calculate a summary relative risk for all smokers (including current and former). 
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+Assuming a single relative risk of mortality due to all causes except for lung cancer among current and former smokers will underestimate the mortality rate for current smokers with high pack-year exposures and former smokers with low years since quitting and overestimate the mortality rate for current smokers with low pack-year exposures and former smokers with high years since quitting. 
+
+Cause Model Diagram
++++++++++++++++++++
+
+There is no cause model diagram for this cause model because it is a mortality-only model. Mortality using should be modeled as follows:
+
+.. math::
+
+	mr_i = (ACMR - CSMR_\text{c426}) * (1 - PAF_\text{*}) * RR_i + EMR(i)_\text{c426}
+
+Where, 
+
+.. list-table:: Definitions
+  :header-rows: 1
+
+  * - Parameter
+    - Definition
+    - Note
+  * - :math:`ACMR`
+    - All cause mortality rate
+    - 
+  * - :math:`mr_i`
+    - Mortality rate for an individual simulant
+    - 
+  * - :math:`CSMR_\text{c426}`
+    - Cause-specific mortality rate for lung cancer
+    - Defined in the :ref:`lung cancer cause model document <2017_lung_cancer>`; use forecasted 2019 value
+  * - :math:`PAF_\text{*}`
+    - Population attributable fraction for smoking on all-cause mortality, excluding lung cancer
+    - Defined below
+  * - :math:`RR_i`
+    - Summary relative risk of mortality due smoking for all causes except lung cancer
+    - Defined below
+  * - :math:`EMR(i)_\text{c426}`
+    - Excess mortality rate of lung cancer for an individual simulant (based on cause model state)
+    - Defined in the :ref:`lung cancer cause model document <2017_lung_cancer>`; use forecasted 2019 value
+
+Data Tables
+++++++++++++++++++++++++++++++++
+
+.. list-table:: Data Sources
+   :header-rows: 1
+   
+   * - Measure
+     - Value/Source
+     - Notes
+   * - :math:`PAF_\text{*}`
+     - :math:`\frac{\sum_{cid}^{n} CSMR_\text{cid} * PAF_\text{cid}}{ACMR - CSMR_\text{c426}}`
```

Comment:
My idea here is that CSMR_c * PAF_c = mortality rate due to smoking for a given cause. Then if we add all of those up across all of the causes and divide by total mortality, it will give us the fraction of all mortality attributable to smoking. Does that sound appropriate to you? I think we are avoiding having to do fancy PAF calculations by transforming with CSMRs and then adding.

Reply:
I think it is fine, because each death has a single underlying cause. But I'm having trouble completely convincing myself.